sanitizer/_boundary_wof.js

```diff
@@ -0,0 +1,38 @@
+const check = require('check-types');
+
+function _sanitize(raw, clean) {
+  // error & warning messages
+  var messages = { errors: [], warnings: [] };
+
+  // target input param
+  var wof_id = raw['boundary.wof'];
+
+  // param 'boundary.wof' is optional and should not
+  // error when simply not set by the user
+  // must be valid string
+  if (check.assigned(wof_id)) {
+    if (!check.nonEmptyString(wof_id)) {
+      messages.errors.push('boundary.wof is not a string');
+    }
+
+    // must be only digits
+    else if ( /^\d+$/.test(wof_id) ) {
```

Comment:
Looks like it's possible for the `wof_id` to be a non-string value here, for instance, it could be an array if someone used the `[]` array URI scheme.

What would be the effect of calling `.test()` on a non-string value?

Reply:
Do you know of a sample query that would result in a non-string value that I could use to test with?  I tried eg: ``boundary.wof[]=420551221`` and `` boundary.wof[0]=420551221&boundary.wof[1]=123 `` but they both get caught by the _single_scalar_parameters sanitizer.

Reply:
Nice. Good to know one of @trescube's first ever contributions long ago is still preventing problems.